conda/recipes/rapids_dask_dependency/meta.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+package:
+- name: rapids_dask_dependency
+  version: 23.10.00.0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask >=2023.7.1
```

Comment:
Another question we might want to consider is how tightly we pin. Do we want an upper bound?

Reply:
During development we never know what the upper bound really is given there's no predictability on Dask features/bugs/release schedule, therefore I think keeping no upper bound as we do currently is more appropriate.